### ISSUE TITLE
How to integrate with angular 5 and above

### ISSUE THREAD

**Issue by vijaynt** [chase-moskal/webp-hero#5]
We have installed the package and add the following in the polyfill

`import 'webp-hero/dist/webp-hero.polyfill';`

But show the following error

**Failed to compile.**

**./node_modules/webp-hero/dist/dwebp.js Module not found: Error: Can't resolve 'fs' in 'D:\\Project_Name\node_modules\webp-hero\dist'**

Advance thanks for the help.

**Comment by chase-moskal** [comment]
@vijaynt 

The error confuses me, because webp-hero does not use the node "fs" module at all

- webp-hero is a browser library with no node dependencies
- the text `"fs"` does not appear in the webp-hero codebase

You may want to try using the polyfill like this in your frontend app:

```js
import WebpHero from "webp-hero"

const webpHero = new WebpHero()

webpHero.polyfill()
```

Good luck my friend, let me know if you have any further troubles

**Comment by chase-moskal** [comment]
@vijaynt, @blahmonkey — the fix for this issue has been published in `webp-hero@0.0.0-dev.21`

fixed in this commit: 4904f32c1957af3ec2f399cb310c3fef8ce725ca

:tada: :+1:
